blueocean-pipeline-editor/src/main/js/components/editor/InputTextArea.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+function cn(...args) {
+    let out = null;
+    for (let i = 0; i < args.length; i++) {
+        if (args[i]) {
+            if (!out) {
+                out = args[i];
+            } else {
+                out += ' ' + args[i];
+            }
+        }
+    }
+    return out;
+}
+
+function autoResizeHeight() {
+    event.target.style.height = 'auto';
```

Comment:
Is `event` automatically in scope here by virtue of this being an event handler? or should it be a method param?

Reply:
yes, it is automatically in scope, no need to set it as a method param